scripts/verify-issue-gates.sh

```
@@ -0,0 +1,116 @@
+else
+  echo "✅ Test files found:"
+  echo "$TEST_FILES" | sed 's/^/   /'
+fi
+
+# Gate 3: Tests Were Written First (check git history)
+echo -e "\n📋 GATE 3: Test-First Verification"
+FIRST_TEST_COMMIT=$(git log --oneline --grep="test.*issue.*$ISSUE_NUMBER" | tail -1)
+FIRST_IMPL_COMMIT=$(git log --oneline --grep="feat.*issue.*$ISSUE_NUMBER" | tail -1)
+
+if [ -n "$FIRST_TEST_COMMIT" ] && [ -n "$FIRST_IMPL_COMMIT" ]; then
+  TEST_DATE=$(git log -1 --format=%at $(echo $FIRST_TEST_COMMIT | cut -d' ' -f1))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)